effect.cpp

```diff
@@ -1730,6 +1765,36 @@ EffectAIIntent Effect::shouldAIApply(const Creature* victim, bool isEnemy) const
             return EffectAIIntent::WANTED;
         return EffectAIIntent::NONE;
       },
+      [&](const Effects::AddAutomatonParts& e) {
+        if(isEnemy) {
```

Comment:
This is a hot function, so the code is going to be way too slow. Do you need AI attaching parts at all?

Reply:
will remove this from this PR until I come up with a more efficient way then instantiating and testing